spec/sparsam_spec.rb

```diff
@@ -31,6 +31,12 @@
       subdata.id_s.should == "id_s default"
     end
 
+    it "respects equality of Sets" do
+      us1 = US.new
+      us2 = US.new
+      expect(Set[us1]).to eq(Set[us2])
```

Comment:
Looks like sparsam is not updated to use `expect` rspec syntax
```suggestion
      Set[us1].should == Set[us2]
```
would hopefully fix the assertion